.gitignore

```diff
@@ -9,3 +9,4 @@ pkg/
 /.yardoc/
 /rdoc/
 /shippable/
+vendor/*
```

Comment:
The preferred approach in the Asciidoctor ecosystem is to store your local gems in .bundle/gems. The .bundle folder is already excluded. I don't want to add another exclusion.